cvmfs/server/cvmfs_server_ingest_tarball.sh

```diff
@@ -313,10 +323,4 @@ cvmfs_server_ingest_tarball() {
   publish_after_hook $name
   publish_succeeded  $name
 
-  # force and health check with repeairs.
-  # it unmount and remount the fs making the new extracted file "appear"
-  # then it checks that everything is ok that doesn't hurt
-  health_check -r $name
```

Comment:
I don't understand why this command does not unmount and remount automatically the repo making the new folders appears

Reply:
healt_check was making also this job